backend/dining/views.py

```diff
@@ -36,6 +41,20 @@ def get(self, request):
         return Response(d.get_venues())
 
 
+class Menus(generics.ListAPIView):
+    """
+    GET: returns list of all menu data for within the week
+    """
+
+    serializer_class = DiningMenuSerializer
+
+    def get_queryset(self):
+        # Return all menus within start_date and end_date
+        start_date = timezone.now().date()
+        end_date = timezone.now().date() + datetime.timedelta(days=6)
```

Comment:
Might be a good idea here to change this to `start_date + datetime.timedelta(days=6)`